collective/deletepermission/tests/base.py

```diff
@@ -27,13 +27,11 @@ def set_local_roles(self, context, user, *roles):
         context.reindexObjectSecurity()
         transaction.commit()
 
-    def get_status_messages(self):
-        request = self.layer['request']
-        messages = [msg.message for msg in IStatusMessage(request).show()]
-        return messages
-
     def get_actions(self):
-        return browser.css('#plone-contentmenu-actions .actionMenuContent a').text
+        if testing.IS_PLONE_5_OR_GREATER:
+            return browser.css('#plone-contentmenu-actions .plonetoolbar-content-action a').text
```

Comment:
We could use the `editbar` page object here -> https://github.com/4teamwork/ftw.testbrowser/blob/master/ftw/testbrowser/pages/editbar.py#L79

Reply:
Oh yes, nice! Updated...